Rakefile

```
@@ -13,15 +13,30 @@ if RUBY_VERSION < "2.6"
   end
   libs = []
 else
-  require 'rake/extensiontask'
-  extask = Rake::ExtensionTask.new(name) do |x|
-    x.lib_dir.sub!(%r[(?=/|\z)], "/#{RUBY_VERSION}/#{x.platform}")
+  if RUBY_ENGINE == 'jruby'
+    require 'rake/javaextensiontask'
+    Rake::JavaExtensionTask.new("wait") do |ext|
+      require 'maven/ruby/maven'
+      ext.source_version = '1.8'
+      ext.target_version = '1.8'
+      ext.ext_dir = 'ext/java'
+      ext.lib_dir = 'lib/io'
+    end
```

`case`...`when` would be better for parallel conditions.
And setting `libs` here can make the next `Rake::TestTask.new(:test)` block unchanged.
```suggestion
when RUBY_ENGINE == 'jruby'
  require 'rake/javaextensiontask'
  Rake::JavaExtensionTask.new("wait") do |ext|
    require 'maven/ruby/maven'
    ext.source_version = '1.8'
    ext.target_version = '1.8'
    ext.ext_dir = 'ext/java'
    ext.lib_dir = 'lib/io'
  end
  libs = ["ext/java/lib", "lib"]
```

---

I will make this change today